### PR TITLE
Adding a link to an example to turn off Cortana

### PR DESCRIPTION
**Pull request by NancyForero**
Adds a link to an example of the powershell command

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/rooms/rooms-manage.md [hunk 1]
```diff
@@ -96,7 +96,7 @@ To learn more about configuring your device using the XML configuration file, se
 At the device level, you can configure Cortana to be activated for:
 - _Push to talk_ using PowerShell  
   - How to turn it on: Cortana _Push to talk_ is enabled by default for all devices that are set up with locale en-us regardless of the geographic region
-  - How to turn it off: use Powershell. 
+  - How to turn it off: use Powershell. [Learn More](https://docs.microsoft.com/en-us/powershell/module/skype/set-csteamscortanapolicy?view=skype-ps#example-1)
```
Comment (JohanFreelancer9) [inline Teams/rooms/rooms-manage.md:99]:
```suggestion
  - How to turn it off: use Powershell. To learn more, see [this example](/powershell/module/skype/set-csteamscortanapolicy?view=skype-ps#example-1).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @NancyForero, kindly commit the suggested change to change to a relative link. Many thanks.

**Comment by NancyForero** [comment]
committed suggested change. thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@dstrome PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by dstrome** [comment]
section removed from topic and replaced with link to https://docs.microsoft.com/en-us/microsoftteams/cortana-in-teams

**Pull request closed without merging**